src/expr/core/src/window_function/kind.rs

```diff
@@ -19,7 +19,7 @@ use crate::aggregate::AggKind;
 use crate::Result;
 
 /// Kind of window functions.
-#[derive(Debug, Display, FromStr, Clone, PartialEq, Eq, Hash)]
+#[derive(Debug, Display, FromStr /* for builtin */, Clone, PartialEq, Eq, Hash)]
```

Comment:
What is "for builtin"

Reply:
We can only parse builtin window functions directly from a string.